config/routes.rb

```diff
@@ -159,8 +159,12 @@
   match 'info/suggested/:key/:value' => 'user_tags#suggested'
   match 'info/tags/create/:id' => 'user_tags#create'
   match 'info/tags/delete/:id' => 'user_tags#delete'
+  match 'info/location/create/:id' => 'location_tags#create'
+  match 'info/privacy' => 'users#privacy'
```

Comment:
I'm curious - what do you think of paths like 'profile/suggested...', 'profile/tags/create/:id' etc -- instead of "info"?